spec/classes/duo_unix_spec.rb

```diff
@@ -35,7 +37,6 @@
           when 'RedHat'
             it { is_expected.to contain_class('duo_unix::yum') }
             it { is_expected.to contain_file('/etc/pki/rpm-gpg/DUO-GPG-PUBLIC-KEY') }
-            it { is_expected.to contain_yumrepo('duosecurity') }
```

Comment:
Why drop the yumrepo test?  Seems like it'd still be useful.

Reply:
The next line in [duplicates](https://github.com/mozilla-it/puppet-duo_unix/blob/master/spec/classes/duo_unix_spec.rb#L39) the check.

Reply:
clearly I excel at reading.  :P